documentation/2-options.md

```diff
@@ -918,6 +918,28 @@ By default, requests will not use [method rewriting](https://datatracker.ietf.or
 
 For example, when sending a `POST` request and receiving a `302`, it will resend the body to the new location using the same HTTP method (`POST` in this case). To rewrite the request as `GET`, set this option to `true`.
 
+### `enableUnixSocket`
+**Type: `boolean`**\
+**Default: `false`**
+
+Set to `true` to enable unix socket domain.
+
+When it is enabled, requests can also be sent via [UNIX Domain Sockets](https://serverfault.com/questions/124517/what-is-the-difference-between-unix-sockets-and-tcp-ip-sockets).\
```

Comment:
Inconsistent pluralization of "socket".

Same in other places too.

Reply:
What does it mean?
Do we wanna rename the option from `enableUnixSocket` to `enableUnixSockets`?

Reply:
> Do we wanna rename the option from enableUnixSocket to enableUnixSockets?

Yes, and also mention of UNIX Sockets should use the plural version.

Reply:
Changed in 7ac2d9cf8fbf17d27677434842a3585f16a81e49